docs/book/src/user/quick-start.md

```diff
@@ -465,6 +489,43 @@ export AZURE_NODE_MACHINE_TYPE="Standard_D2s_v3"
 export AZURE_RESOURCE_GROUP="<ResourceGroupName>"
 ```
 
+{{#/tab }}
+{{#tab CloudStack}}
+
+A ClusterAPI compatible image must be available in your Cloudstack. For instructions on how to build a compatible image
+see [image-builder (Cloudstack)](https://image-builder.sigs.k8s.io/capi/providers/cloudstack.html)
+
+Prebuilt images can be found [here](http://download.cloudstack.org/templates/capi/)
+
+To see all required Cloudstack environment variables execute:
+```bash
+clusterctl generate cluster --infrastructure cloudstack --list-variables capi-quickstart
+```
+
+Apart from the script, the following Cloudstack environment variables are required.
+```bash
+# Set this to the name of the zone in which to deploy the cluster
+export CLOUDSTACK_ZONE_NAME=<zone name>
+# The name of the network on which the VMs will reside
+export CLOUDSTACK_NETWORK_NAME=<network name>
+# The endpoint of the workload cluster
+export CLUSTER_ENDPOINT_IP=<cluster endpoint address>
+export CLUSTER_ENDPOINT_PORT=<cluster endpoint port>
+# The service offering of the control plane nodes
+export CLOUDSTACK_CONTROL_PLANE_MACHINE_OFFERING=<control plane service offering name>
+# The service offering of the worker nodes
+export CLOUDSTACK_WORKER_MACHINE_OFFERING=<worker node service offering name>
+# The capi compatible template to use
+export CLOUDSTACK_TEMPLATE_NAME=<template name>
+# The ssh key to use to log into the nodes
+export CLOUDSTACK_SSH_KEY_NAME=<ssh key name>
+# The kubernetes version of the workload cluster
+export KUBERNETES_VERSION=<kubernetes version>
```

Comment:
nit: You shouldn't have to set the Kubernetes version, clusterctl does that based on `--kubernetes-version`